main.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require_relative 'entities/library'
+
+library = Library.new
```

Comment:
requirements:

> The root file must contain: 
> Library instantiation
> Creating of all basic Entities in several examples
> Adding at least one example of each entity to Library instance
> Call of the “store/save” method
> Call of all 3 statistics methods.
>

Reply:
i tried to do like this:
"library = Library.new
library.generate_data
library.load_data
library.save_data
library.delete_data
library.show"
Should it look differently?

Reply:
please add `library.add_entity` for every entity type

Reply:
can you explain more how to do this?